### PR TITLE
docs - remove filespace from docs

### PR DESCRIPTION
**Pull request by mkiyama**
Replace references of filespace to tablespace.
Update format of gpaddmirror and gpmovemirror config file to remove filespace.

Other misc updates.
system_catalogs/gp_distribution_policy.xml
  - Update policytype description

system_catalogs/pg_database.xml
   - add datminmxid column

Notes
currently gpmovemirrors does not work, reference doc for utility is hidden.

In ddl-tablespace.xml
   - Moving the Location of Temporary or Transaction Files -  does not work

For gpinitstandby the -F option is remove however it can be used as the data directory for the standby master.
Should -F be documented?

Link to HTML format of updated docs on a temporary GPDB review site.

Admin Guide
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/admin_guide/ddl/ddl-tablespace.html
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/admin_guide/highavail/topics/g-enabling-master-mirroring.html
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/admin_guide/highavail/topics/g-enabling-segment-mirroring.html

Best Practices
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/best_practices/ha.html

System Catalog changes
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/ref_guide/system_catalogs/gp_distribution_policy.html
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/ref_guide/system_catalogs/gp_expansion_tables.html
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/ref_guide/system_catalogs/pg_database.html
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/ref_guide/system_catalogs/pg_tablespace.html

GPDB Utilities
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/utility_guide/admin_utilities/gpaddmirrors.html
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/utility_guide/admin_utilities/gpcopy.html
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/utility_guide/admin_utilities/gpinitstandby.html
https://docs-msk-gpdb6-fspace-dev.cfapps.io/600/utility_guide/admin_utilities/gptransfer.html

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/highavail/topics/g-enabling-master-mirroring.xml [hunk 1]
```diff
@@ -18,7 +18,8 @@
       <ol id="ol_x55_3gs_rcb">
         <li id="ki160206">Ensure the standby master host is installed and configured:
             <codeph>gpadmin</codeph> system user created, Greenplum Database binaries installed,
-          environment variables set, SSH keys exchanged, and data directory created.</li>
+          environment variables set, SSH keys exchanged, and data directory, and tablespace
+          directories if needed, are created.</li>
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/highavail/topics/g-enabling-master-mirroring.xml]: and data directory, and tablespace directories if -> and that data directory and tablespace directories, if

File: gpdb-doc/dita/ref_guide/system_catalogs/gp_distribution_policy.xml [hunk 2]
```diff
@@ -25,31 +26,37 @@
...
-                policy.</li></ul></entry>
+                <li><codeph>p</codeph> - Table data is distributed among segment instances.</li>
+                <li><codeph>r</codeph> - Table data is replicated on each segment instance.</li>
+              </ul></entry>
+          </row>
+          <row>
+            <entry colname="col1"><codeph>numsegments</codeph></entry>
+            <entry colname="col2">integer</entry>
+            <entry colname="col3"/>
+            <entry colname="col4">The number of segment instances the table data is distributed
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/system_catalogs/gp_distribution_policy.xml]: the table data is distributed on -> on which the table data is distributed

File: gpdb-doc/dita/ref_guide/system_catalogs/gp_expansion_tables.xml [hunk 3]
```diff
@@ -75,15 +75,36 @@
...
+            <entry colname="col4">Column OIDs for the distribution keys of the table. Value is
+                <codeph>None</codeph> if the table does not have a distribution key column.</entry>
+          </row>
+          <row>
+            <entry colname="col1">
+              <codeph>distribution_policy _type</codeph>
+            </entry>
+            <entry colname="col2">text</entry>
+            <entry colname="col3"/>
+            <entry colname="col4">Table data distribution. Valid values are: <codeph>p</codeph> -
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/system_catalogs/gp_expansion_tables.xml]: make this a bullet list, or maybe just add a matching hyphen "-" after the `r` option.

File: gpdb-doc/dita/best_practices/ha.xml [hunk 4]
```diff
@@ -411,31 +411,24 @@
...
           <li>Run the following query to find the current locations of the primary and mirror
-              segments:<codeblock>SELECT dbid, content, address, port, 
-   replication_port, fselocation as datadir 
-   FROM gp_segment_configuration, pg_filespace_entry 
-   WHERE dbid=fsedbid AND content > -1
-   ORDER BY dbid;</codeblock><p>The
-                <codeph>gp_segment_configuration</codeph>, <codeph>pg_filespace</codeph>, and
-                <codeph>pg_filespace_entry</codeph> system catalog tables contain the current
+              segments:<codeblock>select dbid, content, role, port, hostname, datadir from gp_segment_configuration ;</codeblock><p>The
+                <codeph>gp_segment_configuration</codeph> system catalog table contains the current
```
Comment (skahler-pivotal) [inline gpdb-doc/dita/best_practices/ha.xml]: Noticed you removed the WHERE criteria the removes masters, was that intentional as this mentions earlier it is a list of primary and mirrors

**Pull request closed without merging**